modules/library/main/src/main/java/org/geotools/styling/visitor/RescalingMode.java

```diff
@@ -150,5 +141,17 @@ public Expression rescaleToExpression(Expression scaleFactor, Measure measure) {
                     Measure.ff.literal(measure.uom), scaleFactor, Measure.ff.literal(this));
         }
     }
+    
+    /**
+     * Translates between units and their shortcuts (we can only get the full name from the unit
+     * object
+     */
+    final Map<Unit, String> UNIT_SYMBOLS = new HashMap<Unit, String>() {
+        {
+            put(NonSI.PIXEL, "px");
+            put(NonSI.FOOT, "ft");
+            put(SI.METER, "m");
+        }
+    };
```

Comment:
did this code just change position, or tabs to spaces? I do not see a difference...